eahub/templates/account/signup.html

```diff
@@ -25,4 +25,4 @@
   Read more about our approach to privacy in our <a href="{% url 'privacyPolicy' %}">Privacy Policy</a>.
 </div>
 
-{% endblock %}
+{% endblock %}
```

Comment:
You seem to have accidentally removed a trailing newline here. If your editor is doing this, fix its configuration.